DependencyInjection/LexikTranslationExtension.php

```diff
@@ -193,7 +194,7 @@ protected function buildTranslationStorageDefinition(ContainerBuilder $container
      */
     protected function createDoctrineMappingDriver(ContainerBuilder $container, $driverId, $driverClass)
     {
-        $driverDefinition = new Definition($driverClass, [[realpath(__DIR__.'/../Resources/config/model') => 'Lexik\Bundle\TranslationBundle\Model']]);
+        $driverDefinition = new Definition($driverClass, [[realpath(__DIR__.'/../Resources/config/model') => 'Lexik\Bundle\TranslationBundle\Model'], SimplifiedXmlDriver::DEFAULT_FILE_EXTENSION, true]);
```

Comment:
@raziel057 Can you maybe format this line nicely?

Reply:
I updated this line and another one that was badly formatted. Is it ok for you?

Reply:
Good enough :-)